gax/src/test/java/com/google/api/gax/batching/v2/BatcherImplTest.java

```diff
@@ -255,6 +262,67 @@ public void testWhenThresholdIsDisabled() throws Exception {
     assertThat(result.get()).isEqualTo(4);
   }
 
+  @Test
```

Comment:
Please add javadocs that explain what this test does and that it can't run concurrently with other tests that use Batchers

Reply:
Added explanation for the same with a note.